autogen/safer-cluster/variables.tf.tmpl

```diff
@@ -77,6 +77,12 @@ variable "node_version" {
   default     = ""
 }
 
+variable "release_channel" {
+  type        = string
+  description = "(Beta) The release channel of this cluster. Accepted values are `UNSPECIFIED`, `RAPID`, `REGULAR` and `STABLE`. Defaults to `REGULAR`."
+  default     = "REGULAR"
```

Comment:
Out of curiosity, why pick REGULAR as default over STABLE?

Reply:
See https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/issues/486 & https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/451 for additional context.

Reply:
Right, I looked through it but didn't see any particular reason for picking regular over stable.

While I don't have a strong opinion, based on the GKE documentation I think a *default* setting would be better suited to the more stable option. Users who want newer features etc can switch to regular etc if they would like by overriding. 

I will leave it up to your judgment, just doing a drive-by here. Thanks for the PR, we're hoping to use it as soon as it's released :)

Reply:
I think regular is a reasonable default, because we'd potentially like to add enhanced security features in the future w/o waiting for them to appear in the stable channel. For clusters which require extra stability, they can override.